src/App/Exercises/Js/Try9/TodoForm.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import './styles.css';
+
+const TodoForm = ({ onFormSubmit, onCancel }) => {
+  const [formData, setFormData] = useState({
+    title: '',
+    author: '',
+    note: '',
+  });
+
+  const handleChange = (e) => {
```

Comment:
Podoba mi się to podejście 😄 i fajnie, że poniżej użyłeś sposób z `onSubmit` 👍

Reply:
👍